src/components/message/message-card/MessageCard.tsx

```diff
@@ -47,12 +49,13 @@ export interface RecoveredProps {
 
 interface Props extends OwnProps, RecoveredProps {}
 
-function MessageCardBase({ message, showRaw, showRawHandler }: Props) {
+function MessageCardBase({ message, showRawHandler }: Props) {
 	const messagesStore = useMessagesWorkspaceStore();
 	const selectedStore = useSelectedStore();
 	const workspaceStore = useWorkspaceStore();
 	const { heatmapElements } = useHeatmap();
 	const [isHighlighted, setHighlighted] = React.useState(false);
+	const [messageViewType, setMessageViewType] = React.useState('parsed');
```

Comment:
This should be a string literal type or enum

Reply:
`messageViewType` should be saved with state saver so after changing it and scrolling away from message and returning state wouldn't be default 'parsed'